modelforge/potential/sake.py

```diff
@@ -0,0 +1,510 @@
+from torch._tensor import Tensor
+import torch.nn as nn
+from loguru import logger as log
+from typing import Dict, Type, Callable, Optional, Tuple
+
+from .models import BaseNNP, LightningModuleMixin
+from .postprocessing import PostprocessingPipeline, NoPostprocess
+from .utils import Dense, scatter_softmax
+import torch
+import torch.nn.functional as F
+
+
+class ExpNormalSmearing(torch.nn.Module):
+    def __init__(self, cutoff_lower=0.0, cutoff_upper=5.0, num_rbf=50, trainable=True):
```

Comment:
For the future: we should merge this with the RadialSymmetryFunction class in ANI and SchNET for a common set of Smearing classes.